_third_party/github.com/go-sql-driver/mysql

```diff
@@ -0,0 +1 @@
+Subproject commit 527bcd55aab2e53314f1a150922560174b493034
```

Comment:
Rather than using subprojects we have been using github.com/mjibson/party to copy the source tree into our own repo.